content_scripts/link_hints.js

```diff
@@ -48,6 +48,20 @@ const COPY_LINK_URL = {
     }
   }
 };
+const SHOW_LINK_URL = {
+  name: "link",
+  indicator: "Show link URL",
+  linkActivator(link) {
+    if (link.href != null) {
+      let url = link.href;
+      if (url.slice(0, 7) === "mailto:") { url = url.slice(7); }
+      if (28 < url.length) { url = url.slice(0, 23) + "...."; }
```

Comment:
It's unfortunate that it seems the HUD can't display long messages (see screenshot), and thus must truncate long URLs, which partly defeats the purpose of this feature (which is to review the link URL before you follow it). Is there a way around this?

![image](https://user-images.githubusercontent.com/447442/150869849-cb7a31d6-1568-4acf-97da-a2d764c257e8.png)